<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>405</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>405</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>550</width>
    <height>405</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>550</width>
     <height>405</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>400</width>
     <height>400</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>550</width>
     <height>405</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true"> font-size: 18px;
                    font-weight: bold;
                    padding: 5px;
                    background-color:rgb(245, 245, 255);

                </string>
   </property>
   <property name="title">
    <string/>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QWidget" name="video_player_widget" native="true">
      <property name="minimumSize">
       <size>
        <width>400</width>
        <height>250</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true"> border: 2px solid #34495E;
                                border-radius: 10px;
                                background-color: #000000;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSlider" name="seekSlider">
      <property name="styleSheet">
       <string notr="true">
                                QSlider::groove:horizontal {
                                border: 1px solid #34495E;
                                height: 8px;
                                background: #BDC3C7;
                                border-radius: 4px;
                                }

                                QSlider::handle:horizontal {
                                background: #2980B9;
                                border: 1px solid #34495E;
                                width: 18px;
                                height: 18px;
                                margin: -5px 0;
                                border-radius: 9px;
                                }

                                QSlider::handle:horizontal:hover {
                                background: #3498DB;
                                }

                                QSlider::sub-page:horizontal {
                                background: #2980B9;
                                border: 1px solid #34495E;
                                height: 8px;
                                border-radius: 4px;
                                }
</string>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>80</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border:none;</string>
      </property>
      <property name="title">
       <string/>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>50</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color:red;
                                            border: 1px solid red;
                                            border-radius: 25px;
                                            align-items:center;
                                            padding:8px;</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap>images/buttons/play.png</pixmap>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>50</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color:black;
                                            border: 1px solid black;
                                            border-radius: 25px;
                                            align-items:center;
                                            padding:8px;</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap>images/buttons/pause-black.png</pixmap>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
